app/src/main/java/com/example/trempel/BottomNavExtension.kt

```diff
@@ -0,0 +1,195 @@
+import android.content.Intent
+import android.util.SparseArray
+import androidx.core.util.forEach
+import androidx.fragment.app.FragmentManager
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.navigation.NavController
+import androidx.navigation.fragment.NavHostFragment
+import com.example.trempel.R
+import com.google.android.material.bottomnavigation.BottomNavigationView
+
+fun BottomNavigationView.setupWithNavController(
```

Comment:
can we change it to the `internal` function?

Reply:
done